api/src/test/kotlin/edu/wgu/osmt/collection/CollectionRepositoryTest.kt

```diff
@@ -16,6 +16,7 @@ import edu.wgu.osmt.richskill.RsdUpdateObject
 import edu.wgu.osmt.task.PublishTask
 import edu.wgu.osmt.task.UpdateCollectionSkillsTask
 import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Disabled
```

Comment:
Looks like you are still working in this file. When done, would you please ensure that only related changes are included?